### PR TITLE
[RTM] Hotfix chance DCA labels and descriptions from allowHtml, preserveTag…

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix chance DCA labels and descriptions from allowHtml, preserveTags, decodeEntities

better description for settings https://github.com/MetaModels/attribute_text/issues/11#issuecomment-403275591

### CODE REVIEW

File: contao/languages/en/tl_metamodel_dcasetting.php [hunk 1]
```diff
@@ -58,15 +58,15 @@
...
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][0]               = 'Do not strip html content.';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][0]               = 'Do not encode allowed html tags.';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][1]               =
-    'If you select this, HTML content will be preserved.';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][0]            = 'Do not encode html tags.';
+    'If you select this, allowed HTML tags from system settings will not be encoded.';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][0]            = 'Do not encode all html tags.';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][1]            =
-    'If you select this, HTML tags will not be encoded.';
+    'If you select this, all HTML tags will not be encoded.';
```
Comment (discordier) [inline contao/languages/en/tl_metamodel_dcasetting.php]: Better phrasing: `If you select this, no HTML tags will be encoded.`

File: contao/languages/en/tl_metamodel_dcasetting.php [hunk 2]
```diff
@@ -58,15 +58,15 @@
...
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['decodeEntities'][0]          = 'Decode HTML entities.';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['decodeEntities'][1]          =
-    'Select this, if you want HTML Entities to be decoded.';
+    'If you select this, all HTML entities to be decoded. Note that HTML entities are always decoded if "Do not encode allowed html tags" is true.';
```
Comment (discordier) [inline contao/languages/en/tl_metamodel_dcasetting.php]: `If you select this, all HTML entities will be decoded.`

File: contao/languages/en/tl_metamodel_dcasetting.php (hunk 1 above)
Comment (zonky2) [reply contao/languages/en/tl_metamodel_dcasetting.php]: fixed

File: contao/languages/en/tl_metamodel_dcasetting.php (hunk 2 above)
Comment (zonky2) [reply contao/languages/en/tl_metamodel_dcasetting.php]: fixed

**Pull request merged**